sass/themes/rnd/2017/_rnd17-variables.scss

```diff
@@ -22,4 +22,9 @@ $colour-white: #fff;
 // it the below still needed??
 $item-grey: #ddd;
 /* Single message row background colour, used in June site, to be renamed and updated */
-$cream-grey: #F5F2ED;
\ No newline at end of file
+$cream-grey: #F5F2ED;
+
+/* standard form colors */
+$colour-scorpion-grey:#595959;
+$colour-ribbon-red: #EA0043;
+$colour-persian-purple: #340F78;
```

Comment:
same here as with cr: no need to include them here, by importing the base colour variables they should be available.

Reply:
this break the overall build - it complaining about the rnd

Reply:
That's because the colours.scss file from the base isn't imported in the rnd theme. Don't see a good reason why we shouldn't import it. We should put it back in so we don't have to copy the colour variables from the base theme into the sub theme.